### PR TITLE
Readme improvements / other cleaning

### PR DESCRIPTION
**Pull request by natolambert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
-Currently, the following models are supported:
-* [LLaMA](https://arxiv.org/abs/2302.13971)
+The **fork** adds support for some RLHF methods, such as Direct Preference Optimization (DPO) and 
+Proximal Policy Optimization (PPO) for *some* models. It has been used for models, such as [Tulu 2](https://huggingface.co/allenai/tulu-2-dpo-70b), with more coming soon.
+
+The original `EasyLM` is no longer supported. This is largely supported for research use and doesn't
+come with standard maitenance rules and guidelines.
+
+Currently, the following models are supported to some capacity.
+* [LLaMA](https://arxiv.org/abs/2302.13971), with DPO and PPO
```
Comment (hamishivi) [inline README.md]:
```suggestion
* [LLaMA(-2)](https://arxiv.org/abs/2302.13971), with DPO and PPO
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,59 @@
...
 * [GPT-J](https://huggingface.co/EleutherAI/gpt-j-6B)
 * [RoBERTa](https://huggingface.co/docs/transformers/model_doc/roberta)
+* [OLMo] (coming soon!)
+
+For the core models used, for now just Llama, the directory `EasyLM/models/llama/` contains scripts such as `convert_hf_to_easylm.py` and `convert_easylm_to_hf.py` for easy integration with other libraries. 
+
+Models trained here can also be evalued with [AllenAI's Open Instruct](https://github.com/allenai/open-instruct) repository via `scripts/submit_open_instruct_eval.py`.
```
Comment (hamishivi) [inline README.md]:
```suggestion
Models trained here can also be evaluated with [AllenAI's Open Instruct](https://github.com/allenai/open-instruct) repository via `scripts/submit_open_instruct_eval.py`.
```

File: README.md [hunk 3]
```diff
@@ -82,7 +115,16 @@ If you found EasyLM useful in your research or applications, please cite using t
   url = {https://github.com/young-geng/EasyLM}
 }
 ```
-
+And the citation for this fork specifically, if you wish:
+```
+@software{geng2023easylm,
```
Comment (hamishivi) [inline README.md]:
```suggestion
@software{hamishivi2023easylmfork,
```

**Review by hamishivi — APPROVED** [review]

**Comment by hamishivi** [comment]
LGTM, fixed some minor typos.

**Comment by natolambert** [comment]
Ooops, thx!

**Comment by natolambert** [comment]
(FYI I can't merge yet)

**Comment by hamishivi** [comment]
Ah, I'll do the merge. Added you on the collaborator list so hopefully you can just do it in the future.

**Pull request merged**